source/authentication/use-firebase-auth.txt

```diff
@@ -0,0 +1,189 @@
+
+      #. Click :guilabel:`Save`` and deploy your changes
+
+   .. tab::
+      :tabid: cli
+
+      Run the following command, replacing the value of ``--remote``
+      with your App's Client App ID. This downloads a local copy of your
+      App's latest configuration files and navigates to the
+      configuration file directory, which uses the same name as your
+      App.
+
+      .. code-block:: bash
+
+         realm-cli pull --remote "myapp-abcde"
+         cd myapp
+
+      Add a new Custom JWT authentication provider to your App's
+      ``/auth/providers.json`` file. Use the following configuration as
+      a template, replacing the ``audience`` value with your Firebase
+      Project ID. You can use the provided ``metadata_fields`` as they
+      are or modify the mapping as you wish following the :ref:`Metadata
+      Fields documentation
+      <custom-jwt-authentication-configuration-metadata-fields>`.
+
+      .. literalinclude:: /authentication/custom-jwt-firebase-config-template.json
+         :language: json
+         :caption: /auth/providers.json
+         :emphasize-lines: 7
+
+      Save your changes to ``/auth/providers.json``. Then, push the
+      updated configuration file to deploy your App:
+
+      .. code-block:: bash
+
+         realm-cli push
+
+   .. tab::
+      :tabid: api
+
+      Add a new Custom JWT authentication provider to your App using the
+      :admin-api-endpoint:`Create an authentication provider
+      <tag/adminCreateAuthProvider>` endpoint.
+
+      Use the following configuration as a template. Make sure to:
+
+      - Specify your App's ``$PROJECT_ID`` and ``$APP_ID``
+      - Include an Admin API access token in the ``Authorization``
+        header.
+      - Replace the ``audience`` value in the request body with your
+        Firebase Project ID.
+
+      You can use the provided ``metadata_fields`` as they are or
+      modify the mapping as you wish following the :ref:`Metadata
+      Fields documentation
+      <custom-jwt-authentication-configuration-metadata-fields>`.
+
+      .. literalinclude:: /authentication/custom-jwt-firebase-config-admin-api-template.sh
+         :language: bash
+         :emphasize-lines: 10
+
+Log in with a Firebase JWT
+--------------------------
+
+Once you've configured the Custom JWT authentication provider to use
+Firebase Authentication, you can log in to your App Services App with a
+Firebase JWT.
+
+#. Log the user into Firebase. To learn how, see the relevant
+   :fcm:`Firebase SDK documentation <auth>` for your platform and
+   programming language.
+
+#. Get the user's Firebase JWT. To learn how, see :fcm:`Retrieve ID
+   tokens on clients
+   <auth/admin/verify-id-tokens#retrieve_id_tokens_on_clients>` in the
+   Firebase documentation.
+
+#. Use the Firebase JWT to authenticate with Atlas App Services. You can
+   :ref:`start a session over HTTP <manage-user-sessions>` or log in
+   with an SDK. To learn how, see the docs for your SDK:
```

Comment:
A recent conversation about using more inclusive language has made me try to use "refer to" rather than "see".